src/async.h

```diff
@@ -16,10 +16,10 @@ class Database;
 
 /* abstract */ class AsyncWorker : public Nan::AsyncWorker {
 public:
-  AsyncWorker (
-      leveldown::Database* database
-    , Nan::Callback *callback
-  ) : Nan::AsyncWorker(callback), database(database) { }
+  AsyncWorker(leveldown::Database* database,
+              Nan::Callback *callback,
+              const char *resource_name)
+    : Nan::AsyncWorker(callback, resource_name), database(database) {}
```

Comment:
Bottom line is this. `AsyncWorker` inherits from `Nan::AsyncWorker` and passes `resource_name` down to it. All other workers in `leveldown` are based on `AsyncWorker`.